json-schemas/interface-methods/records-read.json

```diff
@@ -37,8 +36,30 @@
         },
         "recordId": {
           "type": "string"
+        },
+        "protocol": {
+          "type": "string"
+        },
+        "protocolPath": {
+          "type": "string"
+        },
+        "parentId": {
+          "type": "string"
+        }
+      },
+      "$comment": "describe rule requiring either `recordId` OR `protocol` and `protocolPath`",
+      "allOf": [
+        {
```

Comment:
Sanity: the `allOf` outer wrapping doesn't appear to be necessary?

Reply:
Thanks for catching this, I removed it.

I was trying to have the schema validation respond with a message that was more accurate rather than `recordId` is mossing, but it became too verbose and was not much better in messaging.

Simplified here, but replied separately to the testing in another comment.